components/TextFields/examples/TextFieldExample.swift

```diff
@@ -189,6 +189,55 @@ final class TextFieldSwiftExample: UIViewController {
                                                   metrics: nil,
                                                   views: views)
 
+    #if swift(>=4.0)
+      if #available(iOS 11.0, *) {
```

Comment:
I think this should be changed to #if swift(>=3.2) until we move everything to 4.0 as the standard.

https://developer.apple.com/library/content/documentation/Swift/Conceptual/Swift_Programming_Language/Compatibility.html

See the note.

Reply:
This isn't actually checking for Swift. It's the most appropriate of the compiler conditional functions. A check for Swift 4 will ensure this compile only on Xcode 9 and the else will compile on Xcode 8.

Reply:
https://developer.apple.com/library/content/documentation/Swift/Conceptual/Swift_Programming_Language/Statements.html#//apple_ref/doc/uid/TP40014097-CH33-ID538

Reply:
Agree with OP here: Checking Swift >=3.2 will do the same thing and will allow Swift 3 code to use iOS 11 API safely.

Reply:
@codeman7 and @brianjmoore are right. Changing now.